Automattic/wp-calypso
# Remove domain upsell callout code

Related to https://github.com/Automattic/dotcom-forge/issues/7824

> [!WARNING]  
> Merge after https://github.com/Automattic/wp-calypso/pull/92046

## Proposed Changes

* Remove the unused code after https://github.com/Automattic/wp-calypso/pull/92046 is merged

## Why are these changes being made?


* The code will be unused, clean out dead code

## Testing Instructions



* See https://github.com/Automattic/wp-calypso/pull/92046 for testing instructions to make sure nothing is broken

## Pre-merge Checklist
